python/rainbow/simulators/prox_soft_bodies/solver.py

```diff
@@ -1,6 +1,8 @@
 import rainbow.simulators.prox_soft_bodies.mechanics as MECH
 import rainbow.simulators.prox_soft_bodies.collision_detection as CD
-import rainbow.simulators.prox_rigid_bodies.gauss_seidel as GS
+import rainbow.simulators.prox_soft_bodies.prox as PROX
+import rainbow.simulators.prox_soft_bodies.jacobi_parallel as JP
+import rainbow.simulators.prox_soft_bodies.gauss_seidel_parallel as GS
```

Comment:
The GS abbreviation does not quite suggest this is a parallel version.

Reply:
The Jacobi solver got abbreviated by JP, so this should then be GSP to be consistent with JP?

The usage of the letter "P" could be confusing too as these methods are known as projected Gauss-Seidel or project Jacobi or proximal Gauss-Seidel.  Here the acronym PGS is quite well known. Could be easily confused with GSP?

Maybe it would be better to have PAR_GS and PAR_J to make it easier for others to understand what goes on? Or GS_PAR or J_PAR ?

Reply:
It will be changed to PAR_GS or PAR_J.

Reply:
Wonderful